qa/L0_batcher/test.sh

```diff
@@ -42,6 +42,7 @@ VERIFY_TIMESTAMPS=verify_timestamps.py
 DATADIR=${DATADIR:="/data/inferenceserver/${REPO_VERSION}"}
 OPTDIR=${OPTDIR:="/opt"}
 SERVER=${OPTDIR}/tensorrtserver/bin/trtserver
+JETSON=${JETSON:="0"}
```

Comment:
Can you make this more generic. Perhaps at line 37 add something like:
if [ -n "$REPO_ARCH" ]; then
  REPO_VERSION=${REPO_VERSION}_${REPO_ARCH}
fi

And then you shouldn't need any other changes. That same change can be made to all test.sh files.

Reply:
This needs to be done for only TRT and non-ensembles. I will still have to check in the future. But yes I could make it more generic by using REPO_ARCH instead